internal/configs/annotations.go

```diff
@@ -388,6 +388,20 @@ func parseAnnotations(ingEx *IngressEx, baseCfgParams *ConfigParams, isPlus bool
 	return cfgParams
 }
 
+func parseVirtualServerAnnotations(virtualServerEx *VirtualServerEx, enableInternalRoutes bool) bool {
```

Comment:
Usually when a function returns a boolean, the name of the function should be indicative of the boolean value that is being returned. Something like "isInternalRouteServer" or along those lines.